### PR TITLE
Optimize the user doc of real-time wide table join

### PR DESCRIPTION
**Pull request by xuanyu66**
## What is the purpose of the change

Optimize the user doc of real-time wide table join 

## Brief change log

- change mock{value} to real value in case of misunderstanding that mock{value} is a valid reference.
- move the skip constraint and describe the consequence.

## Verifying this change

This change is a trivial user doc change without any test coverage.

## Does this pull request potentially affect one of the following parts:

- Dependencies (does it add or upgrade a dependency): (no)
- The public API: (no)
- The runtime per-record code paths (performance sensitive): (no)

## Documentation

- Does this pull request introduce a new feature? (no)

### CODE REVIEW

File: docs/real-time_wide_table.md [hunk 1]
```diff
@@ -63,44 +62,41 @@ CREATE TABLE `order_wide_table`
 );
 ```
 
-Using flink sql client to sink in `Real-time wide table` scenario. The `mock_${value}` will never be used，just a placeholder:
+Using flink sql client to sink in `Real-time wide table` scenario. You can set an arbitrary value to the column which does not exist in `'tidb.sink.update-columns'`, since they will never be used, just as a placeholder:
```
Comment (Daemonxiao) [inline docs/real-time_wide_table.md:65]: Could you give a link or some context about why setting an arbitrary here? Maybe there is any confusion for the reader in using this function.

Comment (xuanyu66) [reply docs/real-time_wide_table.md:65]: it's under below

Comment (xuanyu66) [reply docs/real-time_wide_table.md:65]: see line 90~93

**Review by zhangyangyu — APPROVED** [review]

**Review by shiyuhang0 — APPROVED** [review]

**Review by humengyu2012 — APPROVED** [review]

**Pull request merged**